worlds/mlss/__init__.py

```diff
@@ -61,6 +63,9 @@ class MLSSWorld(World):
 
     def generate_early(self) -> None:
         self.disabled_locations = set()
+        if self.options.emblems_amount < self.options.emblems_required:
+            self.options.emblems_amount.value = self.options.emblems_required.value
+            logging.warning("Number of emblems required is greater than the number of emblems available. Adjusting value.")
```

Comment:
A couple potential ideas here:
1. You could do this only if the goal is emblem hunt (totally optional)
2. You should include the player's name in the warning so the host knows who this is about
3. You could specify the number it was changed to (totally optional)
```suggestion
        if self.options.goal == "emblem_hunt":
            if self.options.emblems_amount < self.options.emblems_required:
                self.options.emblems_amount.value = self.options.emblems_required.value
                logging.warning(f"{self.player_name}'s number of emblems required is greater than the number of emblems available. "
                                f"Changing to {self.options.emblems_required.value}.")
```